test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperRunnerTest.cs

```diff
@@ -61,7 +62,7 @@ public static TheoryData TagHelperOrderData
             int[] expectedTagHelperOrders)
         {
             // Arrange
-            var runner = new TagHelperRunner();
+            var runner = new TagHelperRunner(new HtmlEncoder());
```

Comment:
what's the issue with using a `TestHtmlEncoder` class right now?  basically just filed aspnet/Mvc#2085 and now extending that problem into the Razor code base instead of writing a 20 line private class at the bottom of this one.